lib/srv/discovery/access_graph.go

```diff
@@ -337,14 +347,16 @@ func (s *Server) initAccessGraphWatchers(ctx context.Context, cfg *Config) error
 			reloadCh := s.newDiscoveryConfigChangedSub()
 			for {
 				// reset the currentTAGResources to force a full sync
-				if err := s.initializeAndWatchAccessGraph(ctx, reloadCh); err != nil {
+				if err := s.initializeAndWatchAccessGraph(ctx, reloadCh); errors.Is(err, errTAGFeatureNotEnabled) {
+					s.Log.Debug("Access Graph specified in config, but the license does not include Teleport Policy. Access graph sync will not be enabled.")
```

Comment:
I don't think that continuing makes sense. The license won't change without a restart, and we will only keep spamming logs.